src/aics-image-viewer/components/App/ChannelUpdater.tsx

```diff
@@ -31,18 +31,19 @@ const ChannelUpdater: React.FC<ChannelUpdaterProps> = ({ index, channelState, vi
     }, [...deps, image, version]);
   };
 
-  useImageEffect(
```

Comment:
Quick question because I'm not as familiar with this repo, why the change from `useImageEffect` to a `useEffect` dependent on `image`?

Reply:
Okay, answered my own question. `useImageEffect` checks for changes to the image but only executes the body if the image is loaded.